services/github/github-commits-since.service.js

```diff
@@ -1,20 +1,14 @@
 'use strict'
 
-const LegacyService = require('../legacy-service')
-const {
-  makeBadgeData: getBadgeData,
-  makeLabel: getLabel,
-} = require('../../lib/badge-data')
-const { makeLogo: getLogo } = require('../../lib/logos')
-const { documentation } = require('./github-helpers')
+const Joi = require('joi')
+const { metric } = require('../text-formatters')
+const { GithubAuthService } = require('./github-auth-service')
+const { fetchLatestRelease } = require('./github-common-fetch')
+const { documentation, errorMessagesFor } = require('./github-helpers')
 
-// This legacy service should be rewritten to use e.g. BaseJsonService.
-//
-// Tips for rewriting:
-// https://github.com/badges/shields/blob/master/doc/rewriting-services.md
-//
-// Do not base new services on this code.
-module.exports = class GithubCommitsSince extends LegacyService {
+const schema = Joi.object().required()
```

Comment:
Is the schema still in progress or do we need to use this?

Reply:
Oops! Fixed.